index.html

```diff
@@ -0,0 +1,335 @@
+            I've also been a good helping hand to a lot of developers in
+            different communities!
+            <br />
+            if you'd like to get in contact about potential job opportunities,
+            please reach out!
+          </p>
+          <a
+            href="https://docs.google.com/document/d/1lYq0Kb50TujzJzDWGobIchRI17JMnAiZMF9OBQgNU68/edit?usp=sharing"
+            class="link"
+            target="_blank"
+            rel="noreferrer noopener"
+          >
+            Get my resume
+          </a>
+          <img class="about-me-icon" src="./media/svg/icon-2.svg" alt="Icon" />
+        </section>
+        <hr />
+        <div class="about-me-section stacks">
+          <ul>
+            <li class="languages list-title">
+              Languages
+              <ul>
+                <li>JavaScript</li>
+                <li>TypeScript</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>Sass</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="frameworks list-title">
+              Frameworks
+              <ul>
+                <li>React & Redux</li>
+                <li>Material-UI</li>
+                <li>Jest</li>
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="skills list-title">
+              Skills
+              <ul>
+                <li>GitHub</li>
+                <li>CodePen</li>
+                <li>GitLab</li>
+                <li>Terminal</li>
+                <li>WSL</li>
+                <li>Remote Pair-Programming</li>
+                <li>Teamwork</li>
+                <li>Mentoring</li>
+                <li>Code Reviews</li>
+                <li>Mobile/Responsive Development</li>
+              </ul>
+            </li>
+          </ul>
+        </div>
+      </div>
+      <div class="form-section">
+        <p>
+          <b class="form-description">
+            I'm always interested in hearing about new projects, so if you'd
+            like to chat please get in touch.</b
+          >
+        </p>
+        <form
+          class="form"
+          action="https://formspree.io/f/maykyqpv"
+          method="POST"
+        >
+          <input
+            type="text"
+            class="user-input"
+            placeholder="Full name"
+            maxlength="30"
+            name="user-name"
+          />
+          <input
+            type="text"
+            class="user-input"
+            placeholder="Email address"
+            name="user-email"
+          />
+          <textarea
+            cols="30"
+            rows="10"
+            class="text"
+            placeholder="Enter text here"
+            name="user-messege"
+          ></textarea>
+          <button type="submit" class="btn">Get in touch</button>
```

Comment:
**[OPTIONAL]** Tip: You may also use an `input` element here with `type` `submit` or simply a `button` element with no `type` attribute (since it defaults to a `submit` type when no `type` attribute is used.)